samples/packages/metadata-impact-report/src/main/kotlin/MetadataImpactReportCfg.kt

```diff
@@ -14,6 +14,8 @@ data class MetadataImpactReportCfg(
     @JsonProperty("include_glossary") val includeGlossary: String = "TRUE",
     @JsonProperty("glossary_name") val glossaryName: String = "Metadata metrics",
```

Comment:
These two `include_glossary` and `glossary_name` should be removed from the `package.pkl` file, too.